pkg/a8rcloud/config.go

```diff
@@ -0,0 +1,22 @@
+package a8rcloud
+
+import (
+	"crypto/x509"
+	"fmt"
+
+	"github.com/telepresenceio/telepresence/rpc/v2/manager"
+)
+
+func certsFromConfig(cfg *manager.AmbassadorCloudConfig) (*x509.CertPool, error) {
+	if cfg.GetProxyCa() == nil {
+		return nil, nil
+	}
+	pool, err := x509.SystemCertPool()
```

Comment:
I guess this should be first in the function so that the `SystemCertPool` is returned as-is if no `proxyCA` is configured? Isn't there a fair chance that the proxy might actually also use a CA that's already in the system pool? If so, then always providing those CAs will make such proxies work without providing an explicit `proxyCA`.

Reply:
so if no pool is returned, the TLS package will just use the system's by default (see [here](https://pkg.go.dev/crypto/tls#Config) under RootCAs) --returning it as is here is basically the same as returning nil.